src/reference/sectionB_elements.inc

```diff
@@ -454,6 +454,8 @@ A :code:`component_ref` element information item (referred to in this specificat
 
       The value of the :code:`component` attribute MUST be a valid component reference, as defined in the section :ref:`Component references<specC_component_reference>`.
```

Comment:
@MichaelClerx are your changes to numbered references in another PR that's waiting, or should we be looking for them here too? Just thinking that this "as defined in" should get a number?

Reply:
They're in https://github.com/cellml/cellml-specification/pull/269

Reply:
So should definitely not be made here or we could get conflicts!